<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-2AFDCIK" failures="5" tests="10" name="Pow Tests" time="0.0" errors="0" timestamp="28 июн. 2018 16:05:33 GMT">
  <testcase classname="com.epam.cdp.CalcUnitTests.Tests.PowCalcTestDouble" name="powNegativeTest" time="0.0">
    <failure type="org.testng.TestException" message="
Method PowCalcTestDouble.powNegativeTest(double, double)[pri:0, instance:com.epam.cdp.CalcUnitTests.Tests.PowCalcTestDouble@42039326] should have thrown an exception of type class java.lang.Exception">
      <![CDATA[org.testng.TestException: 
Method PowCalcTestDouble.powNegativeTest(double, double)[pri:0, instance:com.epam.cdp.CalcUnitTests.Tests.PowCalcTestDouble@42039326] should have thrown an exception of type class java.lang.Exception
at Runners.CalcTestsRunner.main(CalcTestsRunner.java:27)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- powNegativeTest -->
  <testcase classname="com.epam.cdp.CalcUnitTests.Tests.PowCalcTestDouble" name="powNegativeTest" time="0.0">
    <failure type="org.testng.TestException" message="
Method PowCalcTestDouble.powNegativeTest(double, double)[pri:0, instance:com.epam.cdp.CalcUnitTests.Tests.PowCalcTestDouble@42039326] should have thrown an exception of type class java.lang.Exception">
      <![CDATA[org.testng.TestException: 
Method PowCalcTestDouble.powNegativeTest(double, double)[pri:0, instance:com.epam.cdp.CalcUnitTests.Tests.PowCalcTestDouble@42039326] should have thrown an exception of type class java.lang.Exception
at Runners.CalcTestsRunner.main(CalcTestsRunner.java:27)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- powNegativeTest -->
  <testcase classname="com.epam.cdp.CalcUnitTests.Tests.PowCalcTestDouble" name="powTest" time="0.0"/>
  <testcase classname="com.epam.cdp.CalcUnitTests.Tests.PowCalcTestDouble" name="powTest" time="0.0"/>
  <testcase classname="com.epam.cdp.CalcUnitTests.Tests.PowCalcTestDouble" name="powTest" time="0.0"/>
  <testcase classname="com.epam.cdp.CalcUnitTests.Tests.PowCalcTestDouble" name="powTest" time="0.0"/>
  <testcase classname="com.epam.cdp.CalcUnitTests.Tests.PowCalcTestDouble" name="powTest" time="0.0"/>
  <testcase classname="com.epam.cdp.CalcUnitTests.Tests.PowCalcTestDouble" name="powTest" time="0.0">
    <failure type="java.lang.AssertionError" message="Sum operation is incorrect: 2.5^2.5 = 6.25 expected [9.8821176880262] but found [6.25]">
      <![CDATA[java.lang.AssertionError: Sum operation is incorrect: 2.5^2.5 = 6.25 expected [9.8821176880262] but found [6.25]
at com.epam.cdp.CalcUnitTests.Tests.PowCalcTestDouble.powTest(PowCalcTestDouble.java:13)
at Runners.CalcTestsRunner.main(CalcTestsRunner.java:27)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- powTest -->
  <testcase classname="com.epam.cdp.CalcUnitTests.Tests.PowCalcTestDouble" name="powTest" time="0.0">
    <failure type="java.lang.AssertionError" message="Sum operation is incorrect: 2.0^2.5 = 4.0 expected [5.656854249492381] but found [4.0]">
      <![CDATA[java.lang.AssertionError: Sum operation is incorrect: 2.0^2.5 = 4.0 expected [5.656854249492381] but found [4.0]
at com.epam.cdp.CalcUnitTests.Tests.PowCalcTestDouble.powTest(PowCalcTestDouble.java:13)
at Runners.CalcTestsRunner.main(CalcTestsRunner.java:27)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- powTest -->
  <testcase classname="com.epam.cdp.CalcUnitTests.Tests.PowCalcTestDouble" name="powTest" time="0.0">
    <failure type="java.lang.AssertionError" message="Sum operation is incorrect: 2.2^-2.2 = 0.09391435011269719 expected [0.1764696816352779] but found [0.09391435011269719]">
      <![CDATA[java.lang.AssertionError: Sum operation is incorrect: 2.2^-2.2 = 0.09391435011269719 expected [0.1764696816352779] but found [0.09391435011269719]
at com.epam.cdp.CalcUnitTests.Tests.PowCalcTestDouble.powTest(PowCalcTestDouble.java:13)
at Runners.CalcTestsRunner.main(CalcTestsRunner.java:27)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- powTest -->
</testsuite> <!-- Pow Tests -->
